src/hammer-vlsi/defaults.yml

```diff
@@ -56,6 +61,9 @@ vlsi.core:
   # SRAM Generator tool to use.
   sram_generator_tool: "nop"
 
+  # PCB deliverable tool to use.
```

Comment:
Can this documentation clarify a bit on what exactly is meant by the "PCB deliverable tool"? e.g. maybe we could clarify the things we can generate e.g. footprints, etc

Reply:
This is the same text as everything other tool.

Reply:
The first line is fine though it's just not immediately clear to me what "PCB deliverable" exactly means at a high-level glance.

Reply:
I clarified a bit